src/ncp/spinel.h

```diff
@@ -1627,6 +1627,43 @@ typedef enum
      */
     SPINEL_PROP_RADIO_CAPS = SPINEL_PROP_PHY_EXT__BEGIN + 11,
 
+    /// All coex metrics related counters.
+    /** Format: bLt(LLLLLLLL)t(LLLLLLLLL)  (Read-only)
+     *
+     * The contents include three variables and two structs, first one corresponds to
```

Comment:
nit: two variables and two structs?

Reply:
Suggestion: How about adding the common parameters after the two structs? This way we can later add new fields to the common part while keeping backward compatibility... 

Thoughts?

Reply:
Great suggestion. I will update the code.